### PR TITLE
Chore/ps 50 refactor payment in core

### PR DESCRIPTION
**Pull request by sadtuna**
ping @ceigel

### CODE REVIEW

File: lib/payment/merchant_stripe.rb [hunk 1]
```diff
@@ -179,4 +179,72 @@ def fields_needed
...
+      connector.get_charges({}).map { |charge_attrs| new(charge_attrs) }
+    end
+  end
+
+  # Representation of a {Payment} object in the Payment Service.
+  class MerchantPayment < StripeHash
+    attr_accessor :id, :meta, :stripe, :stripe_publishable_key
+
+    class << self
+      def from_tss(profile_id)
```
Comment (ceigel) [inline lib/payment/merchant_stripe.rb]: Could we do the renaming (tss -> payment) in this PR? If not, we could do another one later.

Comment (sadtuna) [inline lib/payment/merchant_stripe.rb]: I'll do this now.

**Comment by sadtuna** [comment]
ping @ceigel

**Comment by sadtuna** [comment]
ping @ceigel

**Comment by ceigel** [comment]
:+1:

**Pull request merged**